static/docs/commands-reference/status.md

```diff
@@ -55,15 +56,32 @@ DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
 differ are listed. For each item listed, either the file name or the checksum is
 shown, and additionally a status word is shown describing the change:
 
-- For the local workspace:
-  - _changed_ means the named file has changed
+- For the changes detected in local workspace:
```

Comment:
it's not clear how this status is connected with the deps/outs statues below. It looks they are two separate lists which is not the case. One of them encloses the other.

Reply:
@shcheklein, I have merged these two lists into the previous heading this should work now. Kindly review the changes.